pkg/scripts/tempesta_installer.sh

```diff
@@ -68,43 +68,16 @@ log() {
     echo "${TIMESTAMP} [${LEVEL}] - ${MESSAGE}" | tee -a $LOGFILE
```

Comment:
we can delete: `declare -r GITHUB_USER="tempesta-tech"` and `declare -r GITHUB_REPO_TEMPESTA="tempesta"`.

Reply:
Done